dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/ds3Panel/DeleteService.java

```diff
@@ -157,20 +155,23 @@ public static void managePathIndicator(final Ds3Common ds3Common,
                 ds3TreeTable.getRoot().getChildren().removeAll(selectedItems);
                 ds3TreeTable.getSelectionModel().clearSelection();
             } else {
-                final TreeItem<Ds3TreeTableValue> selectedItem = ds3TreeTable.getSelectionModel().getSelectedItems().stream()
-                        .findFirst().get().getParent();
-                if (ds3TreeTable.getRoot() == null || ds3TreeTable.getRoot().getValue() == null) {
-                    ds3TreeTable.setRoot(ds3TreeTable.getRoot().getParent());
-                    ds3TreeTable.getSelectionModel().clearSelection();
-                    ds3Common.getDs3PanelPresenter().getDs3PathIndicator().setText(StringConstants.EMPTY_STRING);
-                    ds3Common.getDs3PanelPresenter().getDs3PathIndicatorTooltip().setText(StringConstants.EMPTY_STRING);
-                } else {
-                    ds3TreeTable.setRoot(selectedItem);
-                }
-                ds3TreeTable.getSelectionModel().select(selectedItem);
+                final Optional<TreeItem<Ds3TreeTableValue>> optionalItem = ds3TreeTable.getSelectionModel().getSelectedItems().stream()
+                        .findFirst();
+                optionalItem.ifPresent( item -> {
```

Comment:
Should we do something in the event that it doesn't exist?  Like print a log message, display something?  I.e. What I'm basically asking, if this condition is hit, does it mean there is an error.

Reply:
it just means there's nothing selected in the Ds3Panel, I don't think that's a situation worth logging